infra/examples/gcp-google-workspace/main.tf

```diff
@@ -26,10 +26,9 @@ provider "google" {
   impersonate_service_account = var.gcp_terraform_sa_account_email
 }
 
-
 module "psoxy" {
-  # source = "../../modular-examples/gcp-google-workspace"
-  source = "git::https://github.com/worklytics/psoxy//infra/modular-examples/gcp-google-workspace?ref=v0.4.26"
+#  source = "../../modular-examples/gcp-google-workspace"
+  source = "git::https://github.com/worklytics/psoxy//infra/modular-examples/aws-google-workspace?ref=v0.4.26"
```

Comment:
:red_circle: I don't think this was intended, should be gcp, right?

Reply:
thanks, yeah!